docs/OV_Runtime_UG/auto_device_selection.md

```diff
@@ -1,332 +1,410 @@
+|                         | | MODEL_PRIORITY_LOW                          | | This property is still not fully supported              |
++-------------------------+-----------------------------------------------+-----------------------------------------------------------+
 @endsphinxdirective
 
-
-You can also use the auto-device with limit device choice:
-
 @sphinxdirective
-.. tab:: Package, Docker, open-source installation
-
-   .. code-block:: sh
-
-      ./benchmark_app.py –d AUTO:CPU,GPU –m <model>
-
-.. tab:: pip installation
-
-    .. code-block:: sh
-
-      benchmark_app –d AUTO:CPU,GPU –m <model>
-
+.. dropdown:: Click for information on Legacy APIs 
+
+   For legacy APIs like LoadNetwork/SetConfig/GetConfig/GetMetric:
+   
+   - replace {ov::device:priorities, "GPU,CPU"} with {"MULTI_DEVICE_PRIORITIES", "GPU,CPU"}
+   - replace {ov::hint:model_priority, "LOW"} with {"MODEL_PRIORITY", "LOW"}
+   - InferenceEngine::MultiDeviceConfigParams::KEY_MULTI_DEVICE_PRIORITIES is defined as same string "MULTI_DEVICE_PRIORITIES"
+   - CommonTestUtils::DEVICE_GPU + std::string(",") + CommonTestUtils::DEVICE_CPU is equal to "GPU,CPU"
+   - InferenceEngine::PluginConfigParams::KEY_MODEL_PRIORITY is defined as same string "MODEL_PRIORITY"
+   - InferenceEngine::PluginConfigParams::MODEL_PRIORITY_LOW is defined as same string "LOW"
 @endsphinxdirective
 
-**NOTES:**
-* The default CPU stream is 1 if using `-d AUTO`. 
-* You can use the FP16 IR to work with Auto-device.
-* No demos are fully optimized for Auto-device yet to select the most suitable device, 
-use GPU streams/throttling, and so on.
-
-## Auto-Device Plugin Execution (Python)
+### Device candidate list
+The device candidate list allows users to customize the priority and limit the choice of devices available to the AUTO plugin. If not specified, the plugin assumes all the devices present in the system can be used. Note, that OpenVINO™ Runtime lets you use “GPU” as an alias for “GPU.0” in function calls. 
+The following commands are accepted by the API: 
 
 @sphinxdirective
-.. raw:: html
+.. tab:: C++ API
+
+   .. code-block:: cpp
+
+      /*** With Inference Engine 2.0 API ***/
+      ov::Core core; 
+
+      // Read a network in IR, PaddlePaddle, or ONNX format:
+      std::shared_ptr<ov::Model> model = core.read_model("sample.xml");    
+
+      // Load a network to AUTO using the default list of device candidates.
+      // The following lines are equivalent:
+      ov::CompiledModel model0 = core.compile_model(model);
+      ov::CompiledModel model1 = core.compile_model(model, "AUTO");
+      ov::CompiledModel model2 = core.compile_model(model, "AUTO", {});      
+
+      // You can also specify the devices to be used by AUTO in its selection process.
+      // The following lines are equivalent:
+      ov::CompiledModel model3 = core.compile_model(model, "AUTO:GPU,CPU");
+	   ov::CompiledModel model4 = core.compile_model(model, "AUTO", {{ov::device::priorities.name(), "GPU,CPU"}});
```

Comment:
1. issues with indentation 
2. Use `ov::device::priorities("GPU", "CPU")` instead of `{{ov::device::priorities.name(), "GPU,CPU"}}`